src/theme/default/two-column-layout.m.css

```diff
@@ -4,3 +4,7 @@
 }
```

Comment:
Can you add some comments to these classes please and initial styling so that it works without a theme